src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs

```diff
@@ -245,7 +245,7 @@ private async Task<bool> EnsureCatalogAsync(CancellationToken cancellationToken)
 
             try
             {
-                string json = await _cacheService.GetCatalogAsync(RemoteApiUrl, _cacheFile, cancellationToken).ConfigureAwait(false);
+                string json = await _cacheService.GetContentsFromUriWithCacheFallbackAsync(CatalogUrl, _cacheFile, cancellationToken);
```

Comment:
All the existing `await`ed tasks in this class have `ConfigureAwait(false)`, except the 2 calls to the new methods in `_cacheService`.  Was that an oversight?

Reply:
Yes, it was.  Thanks for catching it!

Reply:
Same applies to these method calls in the `JsDelivrCatalog` and `UnpkgCatalog` classes.